types/text-table/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for text-table v 0.2
+// Project: https://github.com/substack/text-table
+// Definitions by: Saad Quadri <https://github.com/saadq>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Generates borderless text table strings suitable for printing to stdout.
+ */
+declare function table(
+    /** An array of arrays containing strings, numbers, or other printable values. */
+    rows: string[][],
```

Comment:
The comment says `strings, numbers, or other printable values`. Since basically everything has `toString`, is this just a `{}[][]`?

Reply:
Yes, all the examples I had seen only had strings inside it, but there seems to be no issue when using other values as long as they are not `null` or `undefined` so it looks like you are correct.